sdk/automation/Azure.ResourceManager.Automation/src/autorest.md

```diff
@@ -15,16 +15,173 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
+mgmt-debug: 
```

Comment:
Maybe we should remove this configuration after debug.

Reply:
removed